tests/e2e/triggers/workloads.yaml

```diff
@@ -2,10 +2,17 @@ bolt_port: &bolt_port "7687"
 template_cluster: &template_cluster
   cluster:
     main:
-      args: ["--bolt-port", *bolt_port, "--log-level=TRACE"]
+      args: ["--bolt-port", *bolt_port, "--log-level=TRACE", "--storage-properties-on-edges=True"]
       log_file: "triggers-e2e.log"
       setup_queries: []
       validation_queries: []
+storage_properties_edges_false: &storage_properties_edges_false
+  cluster:
+    main:
+      args: ["--bolt-port", "7687", "--log-level=TRACE", "--also-log-to-stderr", "--storage-properties-on-edges=False"]
```

Comment:
Please add a comment to explain why the bolt port is hard-coded (I guess because `pytest_runner.sh` cannot handle different bolt ports).